db/migrate/20240614183935_add_context_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddContextToUsers < ActiveRecord::Migration[7.1]
+  def change
+    add_column :users, :context, :string
```

Comment:
I see that `hangar` uses a separate `account_audits` class for permissions. Didn't think that was necessary to recreate here